### PR TITLE
Lower song length

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: config/permissions.ini [hunk 1]
```diff
@@ -71,7 +71,7 @@ CommandWhitelist = play skip search perms np search help id queue summon
 IgnoreNonVoice = skip search
 ; GrantToRoles =
 ; UserList =
-MaxSongLength = 1200
+MaxSongLength = 1000
```
Comment (adroition) [inline config/permissions.ini]: I think 300 is the one we need

File: config/permissions.ini [hunk 2]
```diff
@@ -106,15 +106,15 @@ CommandWhitelist = np perms help skip play queue
...
 MaxSongs = 8
 AllowPlaylists = yes
 InstaSkip = no
 
 [BotManager]
 ; GrantToRoles =
 UserList = 168993220573790208 188085891548446720
-MaxSongLength = 86400000
+MaxSongLength = 10000
```
Comment (adroition) [inline config/permissions.ini]: Bot manager has infinite length

File: config/permissions.ini [hunk 3]
```diff
@@ -123,7 +123,7 @@ InstaSkip = yes
 [Member]
 GrantToRoles = 191407084170903552
 CommandWhitelist = play skip search perms np search help id queue summon
-MaxSongLength = 2400
+MaxSongLength = 1000
```
Comment (adroition) [inline config/permissions.ini]: Member is made to have double length as default

**Comment by rovertdude** [comment]
mostly change member to something like 300 and i'll merge it

**Pull request merged**